pkg/crc/config/combined_config_test.go

```diff
@@ -17,7 +17,7 @@ const (
 )
 
 func newTestConfig(configFile, envPrefix string) (*Config, error) {
-	storage, err := NewViperStorage(configFile, envPrefix)
+	storage, err := NewCombinedStorage(configFile, envPrefix)
```

Comment:
A storage locker for Combine soldiers? or a CombinedStorage like in a fridge; both cold storage and freezer?